### ISSUE TITLE
Autolink: Link to tweet URL has no context for screen readers

### ISSUE THREAD

**Issue by jamesmacwhite** [twitter/twitter-text#341]
**Is your feature request related to a problem? Please describe.**

Autolinked URLs that lead to the full tweet on the originating Twitter profile lack meaningful context for screen readers to easily determine what the link is, given the link value itself is a https://t.co path.

Example:

This an example tweet with some context that will be then truncated… 
<a class="url" href="https://t.co/ALihCctLBX" rel="external nofollow" target="_blank">https://t.co/ALihCctLBX</a>

**Describe the solution you'd like**

The addition of `aria-label` to better describe the URL and it's purpose when it appears in autolinked tweet data, thus for screen readers clearly indicating what its purpose is.
